waffle-compiler/src/config/inputToConfig.ts

```diff
@@ -16,8 +16,8 @@ export function inputToConfig(input: InputConfig) {
         deprecate('sourcesPath', 'Use sourceDirectory instead.');
         set('sourceDirectory', input[key]);
       } else if (key === 'targetPath') {
-        deprecate('targetPath', 'Use outputDirectory instead.');
-        set('outputDirectory', input[key]);
+        deprecate('targetPath', 'Use compileOutputDirectory instead.');
+        set('compileOutputDirectory', input[key]);
```

Comment:
You should also handle `outputDirectory` configuration option that you are deprecating in this PR.